source/tests/integration/end-to-end/library_classes/StringBuilder.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package java.lang;
+
+public class StringBuilder {
+
+  public StringBuilder append(String str) {
+    return this;
+  }
+}
```

Comment:
Could all changes to library_classes be in a different PR? This causes a lot of changes in integration tests that are not related to the PR, making it hard to review.

Reply:
Thanks, could you rebase this on top of master now that #140 was merged?

Reply:
Done!